Deploy-CAM.ps1

```diff
@@ -905,54 +982,48 @@ function New-PopulatedKeyvault()
 		#generate pfx file from certificate
 		$certPath = $certLoc + '\' + $cert.Thumbprint
 
-		$certPfx = Join-Path $tempDir "self-signed-cert.pfx"
+		if (-not $tempDir) {
+			$tempDir = $env:TEMP
+		}
 
-		#generate password for pfx file
-		$certPswd = -join ((48..57) + (65..90) + (97..122) | Get-Random -Count 16 | % {[char]$_})
-		$secureCertPswd = ConvertTo-SecureString -String $certPswd -AsPlainText -Force
+		$certificateFile = Join-Path $tempDir "self-signed-cert.pfx"
+        if (Test-Path $certificateFile) {
```

Comment:
Yeah that's one reason I wanted to have things happen in a random sub-directory.